apiary.apib

```diff
@@ -792,20 +805,20 @@ Any response that does not provide a ```200 OK``` response code should be treate
 + `targetDate`: "2016-10-01T00:00:00Z" (string, optional) - The target generation date for the recurrence pattern
 + `cronex`: "0 0 12 * * ?" (string, optional) - The cron expression. This is a system generated expression based on the recurrence pattern.
 
-## PmSchedule (object)
-+ `id`: 1 (number, required) - the ID for the schedule object
-+ `title`: `Pm Schedule` (string, required) - the title for the schedule object
+## Base PmSchedule (object)
 + `workOrderTemplateId`: 1 (number, required) - the integer id associated with the work order template for this schedule
 + `recurrencePattern` (Recurrence Pattern, required) - The recurrence pattern
++ `isActive`: 1 (boolean, required) - Boolean value to indicate whether the schedule will continue to generate based on its recurrence pattern
+
+## PmSchedule (Base PmSchedule)
++ `id`: 1 (number, required) - the ID for the schedule object
++ `title`: `Pm Schedule` (string) - the title for the schedule object
```

Comment:
@brianhyder The title is based on the description from the WorkOrderTemplate assigned to the schedule per conversations with Andrew and mockups for the UI